test/integration/features/user_adds_template_to_project.spec.js

```diff
@@ -34,20 +34,22 @@ describe('User adds a template to a project', () => {
           .processTemplate(JSON.stringify(nodeMongoTemplate))
           .then((createFromTemplatePage) => {
             createFromTemplatePage.clickCreate();             // implicit redirect to overview page
-            // verify we have the 2 deployments in the template
-            let deploymentsPage = new DeploymentsPage(project);
-            deploymentsPage.visit();
-            expect(element(by.cssContainingText('td', 'mongodb')).isPresent()).toBe(true); // TODO: use fixture
-            expect(element(by.cssContainingText('td', 'nodejs-mongodb-example')).isPresent()).toBe(true); // TODO: use fixture
-            // verify we have the two services in the template
-            let servicesPage = new ServicesPage(project);
-            servicesPage.visit();
-            expect(element(by.cssContainingText('td', 'mongodb')).isPresent()).toBe(true); // TODO: use fixture
-            expect(element(by.cssContainingText('td', 'nodejs-mongodb-example')).isPresent()).toBe(true); // TODO: use fixture
-            // verify we have one route for the mongo app
-            let routesPage = new RoutesPage(project);
-            routesPage.visit();
-            expect(element(by.cssContainingText('td', 'nodejs-mongodb-example')).isPresent()).toBe(true); // TODO: use fixture
+            return browser.sleep(3000).then(() => {
```

Comment:
@spadgett, I took a guess this would resolve the failing test issue.  Mind taking a look?

Reply:
This seems reasonable. Ideally we would wait for the page to update instead of an arbitrary sleep, but I think it's OK if it unblocks CI.